ceph-salt.spec

```diff
@@ -145,6 +145,7 @@ Salt Formula to deploy Ceph clusters.
 %dir %attr(0755, root, salt) %{fdir}/states/
 %dir %attr(0755, root, salt) %{fdir}/metadata/
 %dir %attr(0755, root, root) %{_datadir}/%{fname}
+%attr(0600, salt, salt) %{buildroot}%{_datadir}/%{fname}/pillar/ceph-salt.sls
 %{fdir}/states/
 %{fdir}/metadata/
 %{_datadir}/%{fname}
```

Comment:
I don't think this spec file change will work. There are two ways to package files and directories under a particular directory: either list them all explicitly, or just list the directory. `ceph-salt-formula` packaging is using the latter technique. You are trying to mix them here.

Reply:
Here are the full contents of this directory:

```
/usr/share/ceph-salt
/usr/share/ceph-salt/pillar
/usr/share/ceph-salt/pillar.conf.example
/usr/share/ceph-salt/pillar/ceph-salt.sls
/usr/share/ceph-salt/pillar/top.sls
```

So, instead of `%{_datadir}/%{fname}` (which just pulls it all in with the default ownership/permissions), we need to explicitly list all the files and directories. Something like this:

```
%dir %attr(0755, root, root) %{_datadir}/%{fname}
%dir %attr(0755, salt, salt) %{_datadir}/%{fname}/pillar
%doc %attr(0644, salt, salt) %{_datadir}/%{fname}/pillar/pillar.conf.example
%attr(0600, salt, salt) %{_datadir}/%{fname}/pillar/ceph-salt.sls
%attr(0644, salt, salt) %{_datadir}/%{fname}/pillar/top.sls
```

and drop the line `%{_datadir}/%{fname}` at the bottom of this `%files -n ceph-salt-formula` section.